src/server/seeds/development/02_messages.js

```diff
@@ -0,0 +1,100 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('channel_messages')
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex('channel_messages').insert([
+        {
+          id: 1,
+          message: 'Hello there!',
+          fk_channel_id: null,
+          fk_user_id: null,
```

Comment:
Lets get the channel seeds and users seed in first, then we can reference those users and channels. Remember we want everything to work when the seeding has been done.

Reply:
I checked the channel-messages is working with users connecting trough fk_user_id"

Reply:
The error is in Migrations / build (pull_request) is because users seeds created by Afruz and I don't have it to be referenced the foreign key . I tried with users in another test repo. It is working.